src/Visitor/Php/Symfony/FormTypeLabelImplicit.php

```diff
@@ -32,6 +32,7 @@ public function enterNode(Node $node)
         $domain = null;
         // use add() function and look at first argument and if that's a string
         if ($node instanceof Node\Expr\MethodCall
+            && (!is_object($node->name) || method_exists($node->name, '__toString')) //fix for issue104
```

Comment:
We do not need this comment. Rather give an example what scenario you are checking for.